catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/LookListFragment.kt

```diff
@@ -0,0 +1,235 @@
+        val items = projectManager.currentSprite.lookList
+
+        adapter = LookAdapter(items)
+        emptyView.setText(R.string.fragment_look_text_description)
+        onAdapterReady()
+    }
+
+    override fun onPrepareOptionsMenu(menu: Menu) {
+        super.onPrepareOptionsMenu(menu)
+        menu.findItem(R.id.catblocks_reorder_scripts).isVisible = false
+        menu.findItem(R.id.catblocks).isVisible = false
+        menu.findItem(R.id.find).isVisible = false
+    }
+
+    override fun packItems(selectedItems: List<LookData?>) {
+        setShowProgressBar(true)
+        var packedItemCnt = 0
+        for (item in selectedItems) {
+            try {
+                BackpackListManager.getInstance().backpackedLooks.add(lookController.pack(item))
+                BackpackListManager.getInstance().saveBackpack()
+                packedItemCnt++
+            } catch (e: IOException) {
+                Log.e(TAG, Log.getStackTraceString(e))
+            }
+        }
+        if (packedItemCnt > 0) {
+            ToastUtil.showSuccess(
+                requireContext(), resources.getQuantityString(
+                    R.plurals.packed_looks,
+                    packedItemCnt,
+                    packedItemCnt
+                )
+            )
+            switchToBackpack()
+        }
+        finishActionMode()
+    }
+
+    override fun isBackpackEmpty(): Boolean =
+        BackpackListManager.getInstance().backpackedLooks.isEmpty()
+
+    override fun switchToBackpack() {
+        val intent = Intent(requireContext(), BackpackActivity::class.java)
+        intent.putExtra(BackpackActivity.EXTRA_FRAGMENT_POSITION, BackpackActivity.FRAGMENT_LOOKS)
+        startActivity(intent)
+    }
+
+    override fun copyItems(selectedItems: List<LookData?>) {
+        setShowProgressBar(true)
+        var copiedItemCnt = 0
+        val currentScene = projectManager.currentlyEditedScene
+        val currentSprite = projectManager.currentSprite
+
+        for (item in selectedItems) {
+            try {
+                adapter.add(lookController.copy(item, currentScene, currentSprite))
+                copiedItemCnt++
+            } catch (e: IOException) {
+                Log.e(TAG, Log.getStackTraceString(e))
+            }
+        }
+        if (copiedItemCnt > 0) {
+            ToastUtil.showSuccess(
+                requireContext(), resources.getQuantityString(
+                    R.plurals.copied_looks,
+                    copiedItemCnt,
+                    copiedItemCnt
+                )
+            )
+        }
+        finishActionMode()
+    }
+
+    private fun disposeItem() {
+        if (Constants.TEMP_LOOK_FILE.exists()) {
+            Constants.TEMP_LOOK_FILE.delete()
+            currentItem = null
+        }
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        disposeItem()
+        val activity: Activity? = requireActivity()
```

Comment:
requireActivity() returns a non-nullable.
```suggestion
        val activity: Activity = requireActivity()
```

Reply:
Ah! trivial mistake, Done!